app/views/widgets/lists.html

```diff
@@ -1,28 +1,26 @@
 {% macro taskAction(prevTaskStatus, status, nextURL, checkYourAnswersURL, content, alreadyDeclared) %}
 
     {% if prevTaskStatus === 'complete' %}
-        {% if status == 'notStarted' %}
-            <!-- Start link -->
-            <p><a href="{{nextURL}}" class="button button-start">{{content.taskNotStarted}}</a></p>
-        {% elif status == 'started' %}
-            <!-- Continue link -->
-            <p><a href="{{nextURL}}" class="button">{{content.taskStarted}}</a></p>
-        {% elif status == 'complete' %}
-            <!-- Check your answers -->
-            {% if alreadyDeclared === 'false' %}
-                <p><a href="{{checkYourAnswersURL}}">{{content.taskComplete}}</a></p>
-            {% else %}
-                <p>{{content.alreadyDeclared}}</p>
-            {% endif %}
-        {% endif %}
-    {% else %}
-        <!-- Previous steps need completing text -->
-        <p class="task-unavailable">{{content.taskUnavailable}}</p>
+        <div class="task-item-content">
```

Comment:
Now the button for each section has been included in the main `.task-item-content` for each section then this <div> can be removed from here